dist/gcc-compatible/Makefile

```diff
@@ -191,7 +191,7 @@ ocamlevercrypt.cma: $(ALL_BINDINGS) $(ALL_ML_STUBS_CMO) $(CTYPES_CMO) $(ALL_C_ST
 	ocamlfind mklib -o ocamlevercrypt $(CTYPES_CMO) $(ALL_C_STUBS) -ccopt -L. -cclib -levercrypt
 
 install-hacl-star-raw: ocamlevercrypt.cmxa ocamlevercrypt.cma libevercrypt.a
-	ocamlfind install hacl-star-raw META $(CTYPES_ML) $(CTYPES_CMX) $(CTYPES_CMO) $(CTYPES_CMI) $(wildcard *.cmxa *.cma *.a *.so *.dll) config.h
+	ocamlfind install hacl-star-raw META $(CTYPES_ML) $(CTYPES_CMX) $(CTYPES_CMO) $(CTYPES_CMI) ocamlevercrypt.cma ocamlevercrypt.cmxa $(wildcard *.a *.so *.dll) config.h
```

Comment:
I suspect this will not work if you invoke "make install-hacl-star-raw" from a clean build because $(wildcard) will be expanded at Makefile read-time and will expand to nothing

example:

```
jonathan@absinthe:/tmp $ cat Makefile 
all:
	echo $(wildcard *.foo)
	touch bar.foo
jonathan@absinthe:/tmp $ make
echo 

touch bar.foo
jonathan@absinthe:/tmp $ make
echo bar.foo
bar.foo
touch bar.foo
```